Classes/Aggregate/NewContentElementWizardAggregate.php

```diff
@@ -92,7 +102,32 @@ protected function process()
 );
 
 EOS
+
         );
+
+        $extensionConfiguration = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['mask_export']);
```

Comment:
There shouldn't be any need to unserialize the extension configuration multiple times. If the icon property is empty, no icon was registered.